app/src/main/java/com/f0x1d/logfox/extensions/context/ContextExtensions.kt

```diff
@@ -132,4 +134,13 @@ fun Context.applyTheme(force: Boolean = false) {
         AppCompatDelegate.setDefaultNightMode(if (nightMode != 0) nightMode else AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM)
 }
 
+fun Context.resolveAttribute(@AttrRes attributeResId: Int): TypedValue? {
+    val typedValue = TypedValue()
+    return if (theme.resolveAttribute(attributeResId, typedValue, true)) typedValue else null
```

Comment:
Please, consider using ` = TypedValue().let { ... }` and `when` instead of `if`. For example:
```kotlin
when (something) {
    true -> ...

    else -> ...
}
```